src/MongoDB/WriteConcern.c

```diff
@@ -66,7 +64,7 @@ PHP_METHOD(WriteConcern, __construct)
 	zend_replace_error_handling(EH_THROW, phongo_exception_from_phongo_domain(PHONGO_ERROR_INVALID_ARGUMENT), &error_handling TSRMLS_CC);
 	intern = (php_phongo_writeconcern_t *)zend_object_store_get_object(getThis() TSRMLS_CC);
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS() TSRMLS_CC, "z|lb!b!", &w, &wtimeout, &journal, &journal_is_null, &fsync, &fsync_is_null) == FAILURE) {
+	if (zend_parse_parameters(ZEND_NUM_ARGS() TSRMLS_CC, "z|lb!b!", &w, &wtimeout, &journal, &journal_is_null) == FAILURE) {
```

Comment:
You would be correct! https://travis-ci.org/mongodb-labs/mongo-php-driver-prototype/jobs/81668599#L1291

I'll change this to `z` and manually check for null values.